parametermanager/api/ParameterManager.Samples/Quickstart.cs

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START parametermanager_quickstart]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.ParameterManager.V1;
+using Google.Protobuf;
+using System.Text;
+
+public class QuickstartSample
+{
+    /// <summary>
+    /// This function demonstrates the quickstart for using the parameter manager using the Parameter Manager SDK for GCP.
```

Comment:
Samples are shown on documentation, each language side by side.  Shouldn't that documentation give this extended explanation instead of each language specific sample?

(And this applies to other PRs as well)